CONTRIBUTING.md

```diff
@@ -0,0 +1,44 @@
+# Contributing
+
+## Overview
+
+This documents explains the processes and practices recommended for contributing enhancements to
+this bundle.
+
+- Generally, before developing enhancements to this charm, you should consider [opening an issue
+  ](https://github.com/canonical/postgresql-bundle/issues) explaining your use case.
+- If you would like to chat with us about your use-cases or proposed implementation, you can reach
+  us at [Canonical Mattermost public channel](https://chat.charmhub.io/charmhub/channels/charm-dev)
```

Comment:
For use-cases discussion, should we change the channel to `https://chat.charmhub.io/charmhub/channels/data-platform`?